app/models/conference.rb

```diff
@@ -329,8 +329,10 @@ def event_distribution
   # * +hash+ -> hash
   def scheduled_event_distribution
     confirmed_events = program.events.where(state: 'confirmed')
-    scheduled_value =  { 'value' => confirmed_events.where.not(start_time: nil).count, 'color' => 'green' }
-    unscheduled_value =  { 'value' => confirmed_events.where(start_time: nil).count, 'color' => 'red' }
+    confirmed_scheduled_events = confirmed_events.select{|event| event.scheduled?}
```

Comment:
But how will we define the scope as the Event model does not have any attribute that defines if the event is scheduled or not?:confused: